conf/messages

```diff
@@ -173,6 +172,11 @@ ssttp.calculator.check-calculation.how-we-calculate-interest = How we calculate
 ssttp.calculator.check-calculation.how-we-calculate-interest.p1 = We only charge interest on overdue amounts.
 ssttp.calculator.check-calculation.how-we-calculate-interest.p2 = We charge the Bank of England base rate plus 2.5%, calculated as simple interest.
 ssttp.calculator.check-calculation.how-we-calculate-interest.p3 = If the interest rate changes during your plan, your monthly payments will not change. If we need to, we''ll settle the difference at the end of the plan.
+ssttp.calculator.check-calculation.cant-afford = I can''t afford the minimum payment
```

Comment:
Is it typo?

Reply:
it's like plays escape character for the apostraphe